MyToolsYourToolsBackend/MyToolsYourToolsBackend.Application/Services/IOfferService.cs

```diff
@@ -8,8 +8,11 @@ namespace MyToolsYourToolsBackend.Application.Services
     public interface IOfferService
     {
         IEnumerable<OfferDto> GetUserOffers(Guid userId);
-        IEnumerable<OfferDto> GetAllOffers();
+        IEnumerable<OfferDto> GetAllOffers(bool onlyActive);
         OfferDto AddOffer(OfferForCreationDto offer, Guid userId);
```

Comment:
Ja rozkminiałem jeszcze o tym, że my te oferty mieliśmy wyświetlać tylko dla grup w których user jest co nie, że ogólnie nie powinien widzieć ofert spoza swoich grup, bo teraz chyba tak jest że je widzi co nie?

Reply:
Tak miało być w zamyśle. Ale potem jak gadaliśmy to, że jednak wszystkie. Mi to jest jedno, to jest mała zmiana

Reply:
No i jeszcze wyświetlanie swoich ofert w liście wszystkich zostawiamy? Czy jednak na tej liście zostawimy tylko te które możemy potencjalnie wypożyczyć? chociaż na razie można zostawić a dopisać do ewentualnych tasków

Reply:
No to moze rzeczywiscie sens by miało wyswietlac tylko z twoich grup

Reply:
A czyje swoje to nie wiem. Moze niech sie grupa wypowie